### PR TITLE
Changed header files, and made a tasks.json for VS Code build task.

### PR DESCRIPTION
**Pull request by chrnigi**
Moved all C standard library includes into stdincludes.h.
Ctrl + Shift + B should now build the PowerHouse.exe in VS Code
Debugging still not working 100%

### CODE REVIEW

File: PowerHouse/Menu.c [hunk 1]
```diff
@@ -71,11 +74,7 @@ int main(void)
 #endif // 0
 
 void clear_terminal() {
-#if _WIN32
-    system("@cls||clear");
-#else
```
Comment (theis999) [inline PowerHouse/Menu.c:76]: Why has this been removed? this makes clear terminal work on in windows using mswc

File: PowerHouse/Menu.c [hunk 2]
```diff
@@ -179,16 +178,16 @@ void appliance_remove_function(void)
...
 
-    printf("appliance removed %s \n", Remove(c) ? "Successfully" : "Unsuccessfully");
+    printf("appliance removed %s \n", _appliance_Remove(c) ? "Successfully" : "Unsuccessfully");
 }
 
 void data_print_function(void)
 {
     int total_rows;
-    Datapoint* data = readCSV("datafiler/DK-DK2_2022_hourly.csv", &total_rows);
+    Datapoint* data = readCSV("PowerHouse/datafiler/DK-DK2_2022_hourly.csv", &total_rows);
```
Comment (theis999) [inline PowerHouse/Menu.c]: This could cause some issues if running the application does not use the same folder.

File: PowerHouse/appliance.h [hunk 3]
```diff
@@ -7,3 +6,15 @@ typedef struct Appliance
     double runTime; // time to run appliance rounded to whole hour
 
 } Appliance;
+
+
+void _appliance_Insert(Appliance a);
+int _appliance_Find(char key[]);
+bool _appliance_Update(Appliance a);
+void _appliance_Upsert(Appliance a);
+bool _appliance_Remove(char key[]);
```
Comment (theis999) [inline PowerHouse/appliance.h]: The prefix underscore is usually reserved for local variables. 
As Appliance seems to be capitalized most other places, it probably should be here too.
So the resulting names should look like "ApplianceRemove()".

File: PowerHouse/csvRead.c [hunk 4]
```diff
@@ -1,15 +1,19 @@
-#include <stdio.h>
-#include <stdlib.h>
-#include <string.h>
-#include <time.h>
+#include "stdincludes.h"
 
 #include "csvRead.h"
+
+
```
Comment (theis999) [inline PowerHouse/csvRead.c]: I am not fan of double empty line

File: PowerHouse/setSettings.c [hunk 5]
```diff
@@ -25,6 +25,13 @@ void setSettings(){
 
     // Saves user input to file
     FILE *settings = fopen("settings.txt", "w");
+    if (settings == NULL)
+    {
+        fprintf(stderr, "Failed to open or locate settings.txt!\n");
+        exit(EXIT_FAILURE);
+    }
+    
+
```
Comment (theis999) [inline PowerHouse/setSettings.c:33]: I am not a fan of double empty line

File: PowerHouse/stdincludes.h [hunk 6]
```diff
@@ -0,0 +1,5 @@
+#include <stdio.h>
+#include <time.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
```
Comment (theis999) [inline PowerHouse/stdincludes.h]: You should include a new_line so github doesn't complain.

**Review by theis999 — CHANGES_REQUESTED** [review]
I am concerned with the removal of system clear, that handles clearning the console.

File: PowerHouse/appliance.h (hunk 3 above)
Comment (chrnigi) [reply PowerHouse/appliance.h]: Done.

File: PowerHouse/stdincludes.h (hunk 6 above)
Comment (chrnigi) [reply PowerHouse/stdincludes.h]: Done.

File: PowerHouse/Menu.c (hunk 2 above)
Comment (chrnigi) [reply PowerHouse/Menu.c]: Changed back, and fixed tasks.json to reflect this.

File: PowerHouse/Menu.c (hunk 1 above)
Comment (chrnigi) [reply PowerHouse/Menu.c:76]: Added back.

**Review by theis999 — APPROVED** [review]

**Pull request merged**